lib/automate_liveness_agent/api_client.rb

```diff
@@ -122,7 +122,7 @@ def wait_for_retry_or_give_up(retry_count)
     def send_request(req)
       if ENV[DEBUG]
         puts "Request Data ".ljust(80, "=")
-        require 'pp'
+        require "pp"
```

Comment:
Heh, for some reason I removed all usage of `pp` but then I forgot to remove this too